NOTES.md

```diff
@@ -107,3 +107,78 @@ Chrome:
 console.assert(false, "robert keeps %s on his balcony", {foo: "bar"})
 Assertion failed: robert keeps %s on his balcony Object {foo: "bar"}
 ```
+
+## console.table - printing of strings
+
+```
+FF:
+
+console.table("the plaice living on the balcony")
+undefined
+console.table():
```

Comment:
No that is the output... I can will add the screenshot for clarification

![screen shot 2016-02-21 at 21 19 34](https://cloud.githubusercontent.com/assets/298166/13205465/d73b423c-d8e0-11e5-974a-f8280db911dc.png)